extensions/wikia/Insights/models/InsightsModel.php

```diff
@@ -9,7 +9,9 @@
 	public $subpage;
 
 	abstract public function getContent( $params );
+	abstract public function initModal();
```

Comment:
Where is this being implemented? This is causing the unit tests to fail with the following fatal error:
`Fatal error: Class InsightsUncategorizedModel contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (InsightsModel::initModal) in extensions/wikia/Insights/models/InsightsUncategorizedModel.php on line 8`